src/constants/preferences.js

```diff
@@ -3,6 +3,10 @@
 // @flow
 
 export default {
+    IGNORE_CHANNEL_MENTIONS_ON: 'ignore_channel_mentions_on',
```

Comment:
Can you make a `src/constants/users.js` and move these into it? This file is specifically for the constants used for `Preference` objects